### PR TITLE
Fixes to Kubernetes/Fedora upstream guide.

### PR DESCRIPTION
**Pull request by aveshagarwal**

### CODE REVIEW

**Comment by eparis** [comment]
avesh works for Red Hat.  CLA is covered.

File: docs/getting-started-guides/fedora/fedora_manual_config.md [hunk 1]
```diff
@@ -94,12 +104,23 @@ done
...
-***We need to configure the kubelet and start the kubelet and proxy***
+***We need to configure the kubelet and proxy and start them.***
+
+**Note:** Due to a bug in kubernetes, we must configure an empty JSON authorization file on each minion.
+
+* Create the JSON file by running the following on all minions.
+
+```
+echo "{}" > /var/lib/kubelet/auth
+```
```
Comment (eparis) [inline docs/getting-started-guides/fedora/fedora_manual_config.md]: This should not be needed upstream.  Only in the old version we shipped in RHEL7.

File: docs/getting-started-guides/fedora/fedora_manual_config.md [hunk 2]
```diff
@@ -108,9 +129,24 @@ KUBELET_PORT="--port=10250"
 # You may leave this blank to use the actual hostname
 KUBELET_HOSTNAME="--hostname_override=fed-minion"
 
+# location of the api-server
+KUBELET_API_SERVER="--api_servers=http://fed-master:8080"
+
 # Add your own!
-KUBELET_ARGS=""
-```       
+KUBELET_ARGS="--auth_path=/var/lib/kubelet/auth"
```
Comment (eparis) [inline docs/getting-started-guides/fedora/fedora_manual_config.md]: Should not be needed....

Comment (aveshagarwal) [inline docs/getting-started-guides/fedora/fedora_manual_config.md]: I just tested without that and it works and the kube cluster comes up.

**Comment by eparis** [comment]
Thanks Avesh.  Lets get rid of the commented out lines in the apiserver config.  They just add confusion.  I'll merge after that.

Next PR, can you fix the kube-proxy systemd unit file, example config, and this guide so the proxy does not have to use KUBE_PROXY_ARGS= and can use something sane?  (contrib/init/system/*)

**Comment by aveshagarwal** [comment]
Eric, i will take care of those things soon.

**Pull request merged**